iidm/iidm-impl/src/test/java/com/powsybl/iidm/network/impl/ShuntCompensatorTest.java

```diff
@@ -25,31 +26,48 @@
 
     private Network network;
     private VoltageLevel voltageLevel;
+    private Terminal terminal;
 
     @Before
     public void setUp() {
         network = NoEquipmentNetworkFactory.create();
+        network.getVoltageLevel("vl2").newLoad()
+                .setId("load")
+                .setBus("busB")
+                .setConnectableBus("busB")
+                .setP0(0)
+                .setQ0(0)
+                .add();
+        terminal = network.getLoad("load").getTerminal();
         voltageLevel = network.getVoltageLevel("vl1");
     }
 
     @Test
     public void baseTest() {
         // adder
         ShuntCompensator shuntCompensator = voltageLevel
-                                        .newShuntCompensator()
-                                            .setId("shunt")
-                                            .setName("shuntName")
-                                            .setConnectableBus("busA")
-                                            .setbPerSection(5.0)
-                                            .setCurrentSectionCount(6)
-                                            .setMaximumSectionCount(10)
-                                        .add();
+                .newShuntCompensator()
```

Comment:
Coding style: I think in the rest of the code, we would put `newShuntCompensator` on the previous line.